guides/release/environment/contents.md

```diff
@@ -3,9 +3,10 @@ A Cardstack environment is the context that Cards run in. By default, it contain
 
 - an application that Cards render in
 - An API that receives requests from the front end, persists changes, and indexes data from external resources
-- An ephemeral data cache so that requests have speedy responses
+- An data cache so that requests have speedy responses
 - Many data plugins for connecting to different kinds of data sources
 - UI interfaces for editing content
+- A backing data persistance layer, such as a git repo or Ethereum.
```

Comment:
We have no ethereum realm yet. The only realms that we have are git-based, filesytem-based, and ephemeral (or tests).